frontend-react/src/App.tsx

```diff
@@ -24,56 +22,48 @@ import { ErrorPage } from "./pages/error/ErrorPage";
 import config from "./config";
 import { USLink } from "./components/USLink";
 import { useScrollToTop } from "./hooks/UseScrollToTop";
-import { EventName, trackAppInsightEvent } from "./utils/Analytics";
-
-const OKTA_AUTH = new OktaAuth(oktaAuthConfig);
+import { trackAppInsightEvent, EventName } from "./utils/Analytics";
+import { getOktaAuth } from "./oktaConfig";
+import { logout } from "./utils/UserUtils";
 
 const { APP_ENV } = config;
+const OKTA_AUTH = getOktaAuth();
 
-initializeSessionBroadcastChannel(OKTA_AUTH); // for cross-tab login/logout
+let sessionStartTime = new Date().getTime();
```

Comment:
Can we defer on these changes until we come to a consensus as a team with Chris and co?

Reply:
Yes, sounds like we could use a conversation on this so I can get aligned. If we don't get to it in 16th min, perhaps the Exp Eng sync on Thurs? @stephenkao